doc-files/release-notes-25.md

```diff
@@ -0,0 +1,155 @@
+[JDK-8349256](https://bugs.openjdk.org/browse/JDK-8349256) | Update PipeWire to 1.3.81 | graphics
+[JDK-8350149](https://bugs.openjdk.org/browse/JDK-8350149) | VBox ignores bias of child controls when fillWidth is set to false | graphics
+[JDK-8351067](https://bugs.openjdk.org/browse/JDK-8351067) | Enforce Platform threading use | graphics
+[JDK-8351867](https://bugs.openjdk.org/browse/JDK-8351867) | No UI changes while iconified | graphics
+[JDK-8353632](https://bugs.openjdk.org/browse/JDK-8353632) | [Linux] Undefined reference to PlatformSupport::OBSERVED_SETTINGS with C++14 | graphics
+[JDK-8353845](https://bugs.openjdk.org/browse/JDK-8353845) | com.sun.javafx.css.BitSet.equals(null) throws NPE | graphics
+[JDK-8354797](https://bugs.openjdk.org/browse/JDK-8354797) | Parent.needsLayoutProperty() should return read-only getter | graphics
+[JDK-8354813](https://bugs.openjdk.org/browse/JDK-8354813) | Parent.isNeedsLayout() may return wrong value in property listener | graphics
+[JDK-8357004](https://bugs.openjdk.org/browse/JDK-8357004) | Windows platform color changes are not picked up in some cases | graphics
+[JDK-8358454](https://bugs.openjdk.org/browse/JDK-8358454) | Wrong \<br> tags in cssref.html  | graphics
+[JDK-8329227](https://bugs.openjdk.org/browse/JDK-8329227) | Seek might hang with fMP4 H.265/HEVC or H.265/HEVC over HTTP/FILE | media
+[JDK-8357714](https://bugs.openjdk.org/browse/JDK-8357714) | AudioClip.play crash on macOS when loading resource from jar | media
+[JDK-8088343](https://bugs.openjdk.org/browse/JDK-8088343) | Race condition in javafx.concurrent.Task::cancel | other
+[JDK-8350048](https://bugs.openjdk.org/browse/JDK-8350048) | Enforce threading restrictions for show and hide methods in Window, Control, and Skin | other
+[JDK-8245602](https://bugs.openjdk.org/browse/JDK-8245602) | Ensemble8: HTMLEditor Toolbar gets scrolled out of view | samples
+[JDK-8146479](https://bugs.openjdk.org/browse/JDK-8146479) | Scene is black after stage is restored (content changed while minimized) | scenegraph
+[JDK-8340322](https://bugs.openjdk.org/browse/JDK-8340322) | Update WebKit to 620.1 | web
+[JDK-8347937](https://bugs.openjdk.org/browse/JDK-8347937) | Canvas pattern test fails and crashes on WebKit 620.1 | web
+[JDK-8349924](https://bugs.openjdk.org/browse/JDK-8349924) | Additional WebKit 620.1 fixes from WebKitGTK 2.46.6 | web
+[JDK-8350284](https://bugs.openjdk.org/browse/JDK-8350284) | WebKit 620.1 crashes on startup on Windows x86 32-bit | web
+[JDK-8351264](https://bugs.openjdk.org/browse/JDK-8351264) | Some images don't load with WebKit 620.1 | web
+[JDK-8352162](https://bugs.openjdk.org/browse/JDK-8352162) | Update libxml2 to 2.13.8 | web
+[JDK-8352164](https://bugs.openjdk.org/browse/JDK-8352164) | Update libxslt to 1.1.43 | web
+[JDK-8353916](https://bugs.openjdk.org/browse/JDK-8353916) | Unexpected event type for DOM mutation events with WebKit 620.1 | web
```

Comment:
There are various issues fixed for WebKit 620.1 while only in this JavaFX version that WebKit version was introduced. Is it really meaningful to specify them?

Reply:
Yes, these should be excluded for the same reason as the ones Michael pointed out above: they are transient bugs that were introduced and fixed in the same version, so will not be seen by anyone migrating from jfx24 to jfx25.

I'll add a `javafx-rn-exclude` label to the 5 follow-on WebKit 620.1 bugs (setting the "introduced-in-release" field won't do it for these, since that would be the update release of the earliest release family).